operations/mimir/autoscaling.libsonnet

```diff
@@ -277,7 +284,7 @@
 
   // newRulerQuerierScaledObject will create a scaled object for the ruler-querier component with the given name.
   // `weight` param works in the same way as in `newQuerierScaledObject`, see docs there.
-  newRulerQuerierScaledObject(name, querier_cpu_requests, min_replicas, max_replicas, weight=1):: self.newScaledObject(name, $._config.namespace, {
+  newRulerQuerierScaledObject(name, querier_cpu_requests, min_replicas, max_replicas, cpu_target_utilization, memory_target_utilization, weight=1):: self.newScaledObject(name, $._config.namespace, {
```

Comment:
I don't think these are actually being used? Should line 301 below be using it? Then are we maintaining `memory_target_utilization` for consistency even though this isn't scaled on memory at all?